packages/flutter_localizations/lib/src/l10n/cupertino_ru.arb

```diff
@@ -0,0 +1,32 @@
+{
+  "todayLabel": "Сегодня",
+  "datePickerHourSemanticsLabelFew": "$hour часа ровно",
```

Comment:
rigth "$hour часа"

Reply:
note these *SemanticLabel* will be read out by iOS voiceover. Are these proposals still correct in that light?

Reply:
by terms of language it it will be correct. I test on Ipad in date and time settings with data picker, voiceover read "10 hours" as "10"  and "10 minutes" as "10 минут". seems strange on hour if compared with minutes

Reply:
Thanks for checking!